images/win/scripts/Installers/Validate-ToolCache.ps1

```diff
@@ -4,63 +4,82 @@
 ##  Desc:  Validate Tool Cache
 ################################################################################
 
-# Python
-$pythonVersions = @(
-    ('2.7.14', 'x86'),
-    ('3.4.4', 'x86'),
-    ('3.5.4', 'x86'),
-    ('3.6.8', 'x86'),
-    ('3.7.2', 'x86')
-    ('2.7.14', 'x64'),
-    ('3.4.4', 'x64'),
-    ('3.5.4', 'x64'),
-    ('3.6.8', 'x64'),
-    ('3.7.2', 'x64')
-)
-
-foreach ($version in $pythonVersions)
-{
-    $v = $version[0]
-    $arch = $version[1]
-    $path = "$env:AGENT_TOOLSDIRECTORY\Python\$v\$arch"
+# Helpers
+function GetChildFolders {
+    param (
+        [Parameter(Mandatory = $True)]
+        [string]$Path
+    )
+    return Get-ChildItem "$Path" | Where-Object {$_.PSIsContainer} | Foreach-Object {$_.Name}

```

Comment:
A small nitpick- Get-ChildItem -Path $path -Directory -Name

Reply:
Thanks, I'll fix it